components/Common/ShellBox/__tests__/index.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+
+import ShellBox from '../index';
+
+Object.assign(navigator, {
+  clipboard: {
+    writeText: jest.fn(),
+  },
+});
+
+const navigatorClipboardSpy = jest.spyOn(navigator.clipboard, 'writeText');
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('ShellBox component', (): void => {
+  it('renders correctly', (): void => {
+    const textToCopy = 'text to be copy';
+    const { container } = render(
+      <ShellBox textToCopy={textToCopy}>mock-children-code</ShellBox>
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it('renders correctly', async () => {
```

Comment:
Why the two tests are "renders correctly"?

Reply:
I don't really understand you question?

Reply:
Both the test cases have the same title.

Reply:
Ho yeah it's from nodejs.dev. I'll change that.